GliaWidgets/Component/AttachmentList/Item/AttachmentSourceItemStyle.swift

```diff
@@ -62,7 +68,9 @@ public class AttachmentSourceItemStyle {
             .first
             .map { titleColor = $0 }
 
-        configuration?.text?.font?.size
-            .map { titleFont = Font.regular($0) }
+        UIFont.convertToFont(
+            font: configuration?.text?.font,
+            textStyle: titleTextStyle
+        ).map { titleFont = $0 }
```

Comment:
@rasmustautsglia @EgorovEI  While it may look convenient to use `map` instead of `if let` to assign a value, I would suggest avoiding it and used `if let` syntax instead. `map` is about transformation, i.e. taking value out of *container*, passing it to pure function and putting returned result back into *container*. This is universal nature of `map`, which can (and should) be applied to any *container* type, like `Result`, Array, Optional or even custom types. Using `.map { titleFont = $0 }` makes it very specific for Optional. So instead I would suggest this:
```
if let font =  UIFont.convertToFont(
    font: configuration?.text?.font,
    textStyle: titleTextStyle
 ) {
    titleFont = font
}
```

Reply:
Since we decided to replace `map` usage with side affects with some another function, I created separated ticket https://glia.atlassian.net/browse/MOB-1617. 
Will update separately from font removal.